platooning_strategic_IHP/include/platoon_manager_ihp.h

```diff
@@ -339,6 +339,8 @@ namespace platoon_strategic_ihp
         bool isCreateGap = false;
         // UCLA: add indicator of lane change 
         bool safeToLaneChange = false;
+
+        int dynamic_leader_index_ = 0;
```

Comment:
Since this variable is an index into a vector, it would be preferred to type it as size_t for consistency.